src/NClient/NClient.Abstractions/Building/INClientOptionalBuilder.cs

```diff
@@ -110,6 +111,22 @@ public interface INClientOptionalBuilder<TClient, TRequest, TResponse> where TCl
         
         #endregion
 
+        #region Host
+        
+        /// <summary>Sets host object.</summary>
+        /// <param name="host">The host of service.</param>
+        INClientOptionalBuilder<TClient, TRequest, TResponse> WithHost(string host);
```

Comment:
For which cases these methods may be needed?

Reply:
Just simplify unnecessary complexity for ppl who prefer simple contracts without any abstractions

Reply:
Okay, good